commands/config.go

```diff
@@ -78,6 +178,69 @@ func configGetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func configSetCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	config, response := c.GetConfig()
+	if response.Error != nil {
+		return response.Error
+	}
+
+	path, err := parseConfigPath(args[0])
+	if err != nil {
+		return err
+	}
+	if err := setConfigValue(path, config, args[1:]); err != nil {
+		return err
+	}
+	if _, res := c.UpdateConfig(config); res.Error != nil {
+		return res.Error
+	}
+
+	printer.Print("Value changed successfully")
```

Comment:
In case of success, I'd suggest to print the whole successful config if the `--format json` is enabled, capturing the return value as `newConfig` (for example) and then printing something like:

```suggestion
	printer.PrintT("Value changed successfully", newConfig)
```

So plain output would just be a confirmation message, and JSON output would yield the newly saved config